srcs/incs/CommonDefinitions.hpp

```diff
@@ -4,6 +4,12 @@
 #include <stdint.h>
 #include <string>
 
+#ifdef DBG
+# define dbg() , __FILE__, __FUNCTION__, __LINE__
+# else
+# define dbg()
+#endif
+
```

Comment:
What do you think about moving this part to `SyscallWrap.hpp`?

Also, the names of the macros were a bit confusing for me because they look like functions (are the parenthesis necessary?). What do you think about renaming them to something like `DEBUG_INFO` and `DEBUG_ARGS`, or something like that?

Reply:
I made it like this because I have seen it somewhere. I thought maybe there might have some problem expanding recusively the macro and I would need some stringify-like trick, but I've been looking into it and there really isn't any problem in doing it without a function-like macro, with just a simple define. Still, I'm gonna look some more into where did I see this macro being named just dbg. I thought it was doxygen (and thus, kind of de facto standard), but I don't know where yet.

Reply:
moved to SyscallWr.hpp